### ISSUE TITLE
getting 404 after added the library when used with apache

### ISSUE THREAD

**Issue by SecretUnicorn** [vishalanandl177/DRF-API-Logger#23]
When I add the lib to the installed apps and to the middleware the wsgi server instance returns only 404. The dev server still works fine though. Any ideas on why this occures ?

**Comment by vishalanandl177** [comment]
I'll look into this ASAP.

**Comment by vishalanandl177** [comment]
It worked fine with the apache server.
Please send me the error logs if available.

**Comment by vishalanandl177** [comment]
Closing due to inactivity.
Feel free to reopen if the issue still persists.

**Comment by SecretUnicorn** [comment]
Ok this reply is very late but I removed the extension back then and forgot about this issue. There is no error in the logs or somewhere. When I add **drf_api_logger** to the installed apps and the _middleware_ the **devserver** works flawlessly. But when I try to access it over wsgi/apache it cant resolve the urls anymore. After I commented out those two lines it worked again.
So the issue must be in the logger somewhere...

![grafik](https://user-images.githubusercontent.com/24612112/135450858-04bc4f1f-3df2-4052-8b44-b6ddf6a31b84.png)
